inc/commoninjectionlib.class.php

```
@@ -408,7 +408,7 @@ public static function getBlacklistedOptions($itemtype)
         if (in_array($itemtype, $CFG_GLPI["networkport_types"])) {
             $raw_options_to_blacklist = array_merge(
                 $raw_options_to_blacklist,
-                NetworkPort::rawSearchOptionsToAdd($itemtype)
+                NetworkPort::rawSearchOptionsToAdd($itemtype) /* @phpstan-ignore-line */
```

Please specify the ignored error in the code comment.